application/CohortManager/src/Functions/ReconciliationService/ReconciliationService.Function/Dockerfile

```
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS base
+
+COPY ./Shared /Shared
+WORKDIR /Shared
+
+RUN mkdir -p /home/site/wwwroot && \
+    dotnet publish ./Common/Common.csproj --output /home/site/wwwroot && \
+    dotnet publish ./Model/Model.csproj --output /home/site/wwwroot && \
+    dotnet publish ./Data/Data.csproj --output /home/site/wwwroot && \
+    dotnet publish ./Utilities/Utilities.csproj --output /home/site/wwwroot && \
+    dotnet publish ./DataServices.Client/DataServices.Client.csproj --output /home/site/wwwroot && \
+    dotnet publish ./DataServices.Core/DataServices.Core.csproj --output /home/site/wwwroot && \
+    dotnet publish ./DataServices.Database/DataServices.Database.csproj --output /home/site/wwwroot
+
+FROM base AS function
+
+COPY ./ReconciliationService /dotnet-function-app
+WORKDIR /dotnet-function-app/ReconciliationService.Function/
```

```suggestion
COPY ./ReconciliationService/ReconciliationService.Functiom /src/dotnet-function-app
WORKDIR /src/dotnet-function-app
```

this matches the pattern for the rest of the dockerfiles

---

This doesn't work as the code in core doesn't get copied.

---

ok fine can you still add /src before /dotnet-function-app so it matches the rest of the project